components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/gen/java/org/wso2/carbon/identity/api/server/identity/governance/v1/IdentityGovernanceApiService.java

```diff
@@ -37,4 +40,6 @@ public interface IdentityGovernanceApiService {
       public Response getPreferenceByPost(List<PreferenceSearchAttribute> preferenceSearchAttribute);
 
       public Response patchConnector(String categoryId, String connectorId, ConnectorsPatchReq connectorsPatchReq);
+
+      public Response patchConnectorsOfCategory(String categoryId, MultipleConnectorsPatchReq multipleConnectorsPatchReq);
```

Comment:
```suggestion
      public Response patchConnectorsCategory(String categoryId, MultipleConnectorsPatchReq multipleConnectorsPatchReq);
```

Reply:
There is already a method called `getConnectorsOfCategory` which handles the GET request of the same endpoint. To be consistent with that, I named this as `patchConnectorsOfCategory`.